### PR TITLE
GenericPolling - added option to additional arguments

### PR DESCRIPTION
**Pull request by IkaDemisto**
## Status
Ready

## Description
GenericPolling playbook can accept additional coma separated arguments and coma separated values that are zipped together (the number of arguments must be equal to number of values). Before the change, the playbook could accept only one argument: `PollingCommandArgName `.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [x] Code Review

## Dependencies
- [x] Scripts:
- [x] ScheduleGenericPolling
- [x] RunPollingCommand
- [x] GenericPollingScheduledTask

Playbooks:
- [x] Scan Site - Nexpose
- [x] Detonate URL - CrowdStrike
- [x] CrowdStrike Falcon Sandbox - Detonate file
- [x] Exchange 2016 Search and Delete
- [x] Enrich DXL with ATD verdict
- [x] Detonate URL - Cuckoo
- [x] Launch Scan - Tenable.sc
- [x] WildFire - Detonate file
- [x] Office 365 Search and Delete
- [x] Detonate File - Cuckoo
- [x] GenericPolling
- [x] Detonate File - Lastline
- [x] Detonate File - BitDam
- [x] Google Vault - Search Mail
- [x] QRadarFullSearch
- [x] Detonate File From URL - JoeSecurity
- [x] Google Vault - Search Drive
- [x] Phishing Investigation - Generic
- [x] Enrich McAfee DXL using 3rd party sandbox
- [x] Google Vault - Display Results
- [x] Detonate URL - ThreatGrid
- [x] Scan Assets - Nexpose
- [x] Tenable.io Scan
- [x] Detonate File - SNDBOX
- [x] ATD - Detonate File
- [x] Detonate URL - McAfee ATD
- [x] Malware Investigation - Generic
- [x] Detonate File - ThreatGrid
- [x] Google Vault - Search Groups
- [x] Detonate URL - Generic
- [x] Detonate URL - JoeSecurity
- [x] Detonate File - Generic
- [x] Detonate File - HybridAnalysis
- [x] Detonate File From URL - WildFire
- [x] Detonate URL - Lastline
- [x] PanoramaCommitConfiguration
- [x] Detonate File - JoeSecurity
- [x] Phishing Playbook - Automated
- [x] Detonate URL - Phish.AI

Test:
- [x] Google_Vault-Search_And_Display_Results_test
- [x] Test-Detonate URL - Phish.AI
- [x] Detonate URL - Generic Test
- [x] Detonate File - HybridAnalysis - Test
- [ ] test_Qradar
- [x] palo_alto_panorama_test_pb
- [x] Detonate File - SNDBOX - Test
- [x] CuckooTest
- [x] Lastline - testplaybook
- [x] tenable-sc-scan-test
- [x] Test Playbook McAfee ATD
- [x] Phishing test - Inline
- [x] Tenable.io Scan Test
- [x] detonate_file_-_generic_test
- [x] Test-Detonate URL - ThreatGrid
- [x] Phishing test - attachment
- [x] Detonate File - Generic Test
- [x] Detonate File - BitDam Test
- [x] JoeSecurityTestDetonation
- [x] No test
- [x] palo_alto_firewall_test_pb
- [x] Detonate File - No Files test
- [x] Test-Detonate URL - Crowdstrike

### CODE REVIEW

**Comment by Itay4** [comment]
@IkaDemisto what is the status here?

File: Playbooks/playbook-GenericPolling.yml [hunk 1]
```diff
@@ -225,5 +233,15 @@ inputs:
   value: {}
   required: true
   description: DT filter for polling IDs. Polling will stop when no results are returned.
+- key: AdditionalPollingCommandArgNames
+  value: {}
+  required: false
+  description: Commas separated arguments of the polling command
```
Comment (BenJoParadise) [inline Playbooks/playbook-GenericPolling.yml]:
```suggestion
  description: Names of additional arguments for the polling command (e.g. arg1,arg2,...)
```

File: Playbooks/playbook-GenericPolling.yml [hunk 2]
```diff
@@ -225,5 +233,15 @@ inputs:
...
+- key: AdditionalPollingCommandArgValues
+  value: {}
+  required: false
+  description: Commas separated  arguments values of the polling command
```
Comment (BenJoParadise) [inline Playbooks/playbook-GenericPolling.yml]:
```suggestion
  description: Values of the additional arguments for the polling command (e.g. value1,value2,...)
```

File: Scripts/script-GenericPollingScheduledTask.yml [hunk 3]
```diff
@@ -164,5 +171,14 @@ args:
   required: true
   description: The tag of the blocking manual task ("Wait For Polling Task To Finish")
   defaultValue: polling
+- name: additionalPollingCommandArgNames
+  description: Commas separated arguments of the polling command
```
Comment (BenJoParadise) [inline Scripts/script-GenericPollingScheduledTask.yml]:
```suggestion
  description: Names of additional arguments for the polling command (e.g. arg1,arg2,...)
```

File: Scripts/script-ScheduleGenericPolling.yml [hunk 4]
```diff
@@ -89,5 +92,14 @@ args:
   required: true
   description: The tag of the blocking manual task ("Wait For Polling Task To Finish")
   defaultValue: polling
+- name: additionalPollingCommandArgNames
+  description: Commas separated arguments of the polling command
+  isArray: true
+- name: additionalPollingCommandArgValues
+  description: Commas separated arguments values of the polling command
```
Comment (BenJoParadise) [inline Scripts/script-ScheduleGenericPolling.yml]:
```suggestion
  description: Values of the additional arguments for the polling command (e.g. value1,value2,...)
```

File: Scripts/script-ScheduleGenericPolling.yml [hunk 5]
```diff
@@ -89,5 +92,14 @@ args:
   required: true
   description: The tag of the blocking manual task ("Wait For Polling Task To Finish")
   defaultValue: polling
+- name: additionalPollingCommandArgNames
+  description: Commas separated arguments of the polling command
```
Comment (BenJoParadise) [inline Scripts/script-ScheduleGenericPolling.yml]:
```suggestion
  description: Names of additional arguments for the polling command (e.g. arg1,arg2,...)
```

File: Tests/conf.json [hunk 6]
```diff
@@ -1194,7 +1194,11 @@
             "playbookID": "CheckDockerImageAvailableTest"
         },
         {
-            "playbookID": "ExtractDomainFromEmailTest"
+          "playbookID": "ExtractDomainFromEmailTest"
```
Comment (BenJoParadise) [inline Tests/conf.json]: Return this line to the proper alignment.

**Comment by IkaDemisto** [comment]
@Itay4 Ready. Need to force merge.

**Pull request merged**